packages/react/src/VideoPlayer/VideoPlayer.module.css

```diff
@@ -42,32 +43,25 @@
   left: 0;
   width: 100%;
   height: 100%;
-  background: transparent;
+  background: #00000033;
```

Comment:
Can we add a TODO comment on this hex value, indicating that we should switch to token once v8 migration is complete

Reply:
👍 0b67ffb8e05b4096a9abab5a10512585434064eb